doc/plugin_agent_workloadattestor_unix.md

```diff
@@ -0,0 +1,6 @@
+# Agent plugin: workloadattestor-unix
+
+The `workloadattestor-unix` plugin attests a workload based on its PID and the UID associated with
```

Comment:
Does each plugin allow for specific selectors (eg. UID) to be used for SPIFFE IDs? If so, it would be good to specify which selectors are enabled here (and in the other plugins).

Reply:
Each plugin only supports one selector, in this case UID. Good point though, should give an example of what a selector would look like.

Reply:
Hm we don't have any docs on using the registration API at all... maybe in a later PR

Reply:
Ok. Filed here. https://github.com/spiffe/spire/issues/298